rmgpy/molecule/resonanceTest.py

```diff
@@ -1419,3 +1419,11 @@ def test_surface_o(self):
 1 X u0 p0 c0 {2,D}
 2 O u0 p2 c0 {1,D}"""))
         self.assertEquals(len(mol_list), 1)
+
+    def sulfur_triple_bond(self):
```

Comment:
I think tests have to start with `test` in order to be recognized and run as a test.